packages/protocol/test/unit/protocol/message_layer/StreamMessage.test.ts

```diff
@@ -51,7 +51,7 @@ describe('StreamMessage', () => {
             assert.strictEqual(streamMessage.encryptionType, EncryptionType.NONE)
             assert.strictEqual(streamMessage.groupKeyId, null)
             assert.deepStrictEqual(streamMessage.getContent(), content)
-            assert.strictEqual(streamMessage.getSerializedContent(), JSON.stringify(content))
+            assert.strictEqual(binaryToUtf8(streamMessage.getSerializedContent()), JSON.stringify(content))
```

Comment:
Use simpler assertions by using `expected` as-is and calculating the `actual`

(also at line 78)